requirements.txt

```diff
@@ -1 +1,3 @@
-
+Django==1.11.3
+Pillow==4.2.1
```

Comment:
You should install Pillow because it allows uploading of images of through the admin page.